### DISCUSSION TITLE
Wysyłanie produktu

### DISCUSSION THREAD

**Discussion by ravitvv** [allegro/allegro-api#7162] [Suggestions - rejected]
Proszę mi wyjaśnić proces uzyskiwania dla niego wszystkich typów kategorii i parametrów, próbuję połączyć Allegro z moim niestandardowym ERP

{
    "errors": [
        {
            "code": "UnableToCreateProductWithoutAllRequiredParametersException",
            "message": "Unable to create product without proper values in all required parameters: [Wydawnictwo, Język] in category: 79157",
            "details": null,
            "path": "parameters",
            "userMessage": "Nie można stworzyć produktu bez podania poprawnych wartości wszystkich parametrów wymaganych: [Wydawnictwo, Język]"
        }
    ]
}

**Comment by PrzemyslawLukanowski** [comment]
Przeniosłem Twój wątek do zakładki Issues (#7163), ponieważ w Discussions zbieramy wyłącznie sugestie, które dotyczą rozwoju API Allegro. Tam udzielę odpowiedzi, natomiast ten watek usuwam.
